react/components.md

```diff
@@ -0,0 +1,96 @@
+### Introduction
+
+Introduction to React components, what they do, and how to write them. In this lesson we'll be going over the basics of React components - what they do, and how to write them. Make sure to use the project you set up in the [Setting Up A React Environment lesson](https://github.com/TheOdinProject/top-meta/issues/221), but try not to copy and paste any code while you're coding along. 
+
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+*   What are components?
+*   How are components created?
+*   Where do components live?
+
+### What are components
```

Comment:
Headings need to be [Wikipedia style cased according to our Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#headings). Can you change headings to adhere to the proper style?